api/src/main/java/net/kyori/adventure/text/ScoreComponent.java

```diff
@@ -27,7 +27,14 @@
 import org.checkerframework.checker.nullness.qual.Nullable;
 
 /**
- * A scoreboard score component.
+ * A component that can display a players score on a scoreboard objective, with an optional fallback value if the search for the score fails.
+ * 
+ * <p>This component consists of:</p>
+ * <ul>
+ *   <li>name - a player username</li>
+ *   <li>objective - a scoreboard objective</li>
+ *   <li>value(optional) - a fallback value to be used if the search fails</li>
```

Comment:
You should mention that the `value` field is no longer present in 1.16, and will therefore be ignored.